egs/mls_english/ASR/zipformer/streaming_decode.py

```
@@ -0,0 +1,900 @@
+        help="The number of streams that can be decoded parallel.",
+    )
+
+    add_model_arguments(parser)
+
+    return parser
+
+
+def get_init_states(
+    model: nn.Module,
+    batch_size: int = 1,
+    device: torch.device = torch.device("cpu"),
```

Ruff (0.12.2)

Do not perform function call `torch.device` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)